.github/workflows/tag.yml

```diff
@@ -1,5 +1,5 @@
 name: Tag
-on: workflow_dispatch
+on: workflow_call
```

Comment:
@pshapiro4broad had a [good suggestion](https://github.com/DataBiosphere/java-pfb/pull/3#discussion_r1284519289) on my PR that we should define the secrets required in the action. It would probably be good to decide on a consistent pattern here!